src/app/state.service.ts

```diff
@@ -29,43 +35,50 @@ export class StateService {
       .slice(0, 30); // always limit to 30 results
   });
 
-  location = inject(Location);
   platformLocation = inject(PlatformLocation);
```

Comment:
suggestion: you can use `ActivatedRoute` instead to initialize the `queryParams`

Reply:
I keep getting `null` with that. Subscribing with `pipe(filter(v => !!v), take(1))` gives a delayed value.